src/cmd/script.rs

```diff
@@ -864,6 +871,13 @@ fn emit_vivado_tcl(
             .iter()
             .map(|t| (format!("TARGET_{}", t.to_uppercase()), None)),
     );
+    if let Some(define) = matches.values_of("defines") {
+        defines.extend(
+            matches.values_of("defines")
+            .unwrap()
+            .map(|t| (t.to_string(), None))
```

Comment:
```suggestion
            define.map(|t| (t.to_string(), None))
```
should work here as well.

Reply:
Done.